css/style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  font-family: futura lt;
+  font-size: 16px;
+  line-height: 1.5;
+  margin: 30px;
+  padding: 60px;
+  background: #EEE;
+}
+
+a {
+  color: #c6a149;
+  text-decoration: none;
+}
+
+strong {
+  font-family: didot;
+  font-size: 20px;
+  text-transform: capitalize;
+  letter-spacing: 0px;
+}
+
+header {
+  text-align: center;
+  margin-bottom: 60px;
+  text-transform: uppercase;
+}
+
+nav {
+  font-size: 15;
```

Comment:
when you are setting font-sizes you must specify the type. For example `rem`, `em` or `px` 🔨